script/parse-examples

```diff
@@ -2,12 +2,23 @@
 
 set -e
 
+cd "$(dirname "$0")/.."
+
+repo=examples/npm
+
+if [ -d "$repo" ]; then
+  pushd "$repo" && git pull
```

Comment:
I was thinking maybe we should do `git reset --hard <SOME SPECIFIC SHA>` so that the build won't ever start breaking spontaneously because of changes they make on the master branch of `npm`. Thoughts?